### PR TITLE
Removes blob inheritance from OriginalXmlLabel

### PR DESCRIPTION
**Pull request by AustinSanders**
Removes blob inheritance from OriginalXmlLabel

## Related Issue
None

## Motivation and Context
More clearly separates functionality so that we can move toward building pvl / blob / functionality independently.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp [hunk 1]
```diff
@@ -30,10 +29,20 @@ namespace Isis {
...
+  }
+
+  /**
+   * Constructs an OriginalXmlLabel from a blob
+   *
+   * @param blob Blob from which to create the OriginalXmlLabel
+   */
+  OriginalXmlLabel::OriginalXmlLabel(Isis::Blob &blob) {
+    fromBlob(blob);
   }
```
Comment (acpaquette) [inline isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp:45]: We almost don't need this blob constructor with how new this class is

File: isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp [hunk 2]
```diff
@@ -20,8 +20,7 @@ namespace Isis {
   /**
    * Constructors a default OriginalXmlLabel with an empty label.
    */
-  OriginalXmlLabel::OriginalXmlLabel() : Isis::Blob("IsisCube", "OriginalXmlLabel") {
-    p_blobPvl += Isis::PvlKeyword("ByteOrder", "NULL");
```
Comment (acpaquette) [inline isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp:24]: We may still want this in the default constructor but I can't give a good reason why 😆

**Review by acpaquette — APPROVED** [review]
Overall looks good! :+1:

File: isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp (hunk 2 above)
Comment (AustinSanders) [reply isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp:24]: I kinda tried, but I wasn't sure how to make that happen... p_blobPvl is a member of Blob, and there's no associated Blob at the point where this is called.  Do you have any ideas for how to make that work?

Comment (acpaquette) [reply isis/src/base/objs/OriginalXmlLabel/OriginalXmlLabel.cpp:24]: I think we want to move this to the toBlob method. Updating the blob label that we return the blob form of the xmllabel. Or do we only need the NULL byteorder when we are reading?

**Pull request merged**